.github/workflows/doc-build.yml

```diff
@@ -168,12 +168,20 @@ jobs:
           zip -r docs.zip docs
 
       - name: Upload docs
+        id: upload
         if: (success() || failure()) && steps.copy.outcome == 'success'
         uses: actions/upload-artifact@v3
         with:
           name: docs
           path: docs.zip
 
+      - name: Remove docs
+        if: (success() || failure()) && steps.upload.outcome == 'success'
+        run: |
+          set -ex
+          # We remove previous docs directory to save space on runner device
+          rm -rf docs/
```

Comment:
Simpler to do that right after line 168

Reply:
The "Upload docs" needs `docs/` directory and its contents. No?

Reply:
It's uploading the zip file

Reply:
Ah, the zip file is not in the directory. 

Then perhaps removing the zip file as well would help. What do you think?

Reply:
We remove `docs.zip` file too.